x-pack/test/api_integration/apis/cases/index.ts

```diff
@@ -5,10 +5,33 @@
  * 2.0.
  */
 
+import {
+  createUsersAndRoles,
+  deleteUsersAndRoles,
+} from '../../../cases_api_integration/common/lib/authentication';
+
+import { loginUsers } from '../../../cases_api_integration/common/lib/utils';
+import { casesAllUser, obsCasesAllUser, secAllUser, users } from './common/users';
+import { roles } from './common/roles';
 import { FtrProviderContext } from '../../ftr_provider_context';
 
-export default function ({ loadTestFile }: FtrProviderContext) {
+export default function ({ loadTestFile, getService }: FtrProviderContext) {
   describe('cases', function () {
+    const supertestWithoutAuth = getService('supertestWithoutAuth');
+
+    before(async () => {
+      await createUsersAndRoles(getService, users, roles);
+      await loginUsers({
```

Comment:
This is to create the user profiles correct?

Reply:
Yep exactly, I refactor this call in a follow up PR here too: https://github.com/elastic/kibana/pull/137532/files#diff-a0c3e734b457a0cba0c28ebe9b18e446ab60252af85817cd429fa81321e8102dR108

Ignore the name, it should be `activate...`